salt/beacons/inotify.py

```diff
@@ -149,19 +158,20 @@ def beacon(config):
 
         beacons:
```

Comment:
I think some of the documentation changes can be ported into `2017.7`, as I noticed several inconsistencies, e.g.: https://docs.saltstack.com/en/develop/ref/beacons/all/salt.beacons.inotify.html
Happy to help with that, just wanted to mention it. Thanks for all this work @garethgreenaway - great stuff!